psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -3739,7 +3739,7 @@ void FISAPT::find() {
         // => JK Object <= //
 
         // TODO: Account for 2-index overhead in memory
-        int nso = primary_->nbf();
+        long int nso = primary_->nbf();
```

Comment:
`size_t` not `long int`, right?

Reply:
I agree with lori here.  Better still:
```suggestion
        auto nso = primary_->nbf();
```